tests/FlexBridgeRabbitMqTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+declare(strict_types=1);
+
+namespace Platformsh\FlexBridge\Tests;
+
+use PHPUnit\Framework\TestCase;
+
+use function Platformsh\FlexBridge\mapPlatformShEnvironment;
+
+class FlexBridgeRabbitMqTest extends TestCase
+{
+    protected $relationships;
+    protected $defaultValues;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->relationships = [
+            'rabbitmqqueue' => [
+                [
+                    'scheme'   => 'amqp',
+                    'username' => 'guest',
+                    'password' => 'guest',
+                    'host'     => 'rabbitmq.internal',
+                    'port'     => 5672,
+                    'path'     => null,
+                    'query'    => [],
```

Comment:
vhost is missing

Reply:
It's only included on Dedicated environments, actually.  Hence the default of `%2f` hard coded into the formatter.